src/Oceananigans.jl

```diff
@@ -142,18 +141,31 @@ import Base:
     push!
 
 "Boolean denoting whether AMGX.jl can be loaded on machine."
-const hasamgx = @static Sys.islinux() ? true : false
+# TODO: find a way to check whether the libraries for AMGX and NETCDF 
+# (libamgxsh and libnetcdf, respectively) are installed on the machine
+const hasamgx   = false #@static Sys.islinux() ? true : false
+const hasnetcdf = false

```

Comment:
why is this line needed?

Reply:
NetCDF_jll isn't available for power 9, so sometimes it's a pain to install. This let's us use Oceananigans even if we don't have working NetCDF

Reply:
What about all the output writer tests? We won’t have them?

Reply:
We can't merge with `hasnetcdf=false`. Instead we need to figure out how to determine whether the user has a working version of `netcdf`, ie

```julia
const hasnetcdf =
    try
        using NCDatasets
        true
    catch
        false
    end
```

If `NetCDF_jll` isn't available for a user's platform, they have to install it manually somehow (either by finding a binary via miniconda or another service, or by using the instructions provided by NetCDF). They shouldn't have to modify Oceananigans source code (but they may have to re-build it, if they have already compiled it and at that time `hasnetcdf=false`).

Reply:
would

```Julia
const hasnetcdf =
    try
        using NCDatasets
        true
    catch
        false
    end
```

be OK for what you are running on Satori?  Shall we do this for now?